index.js

```diff
@@ -1 +1,72 @@
-// Your code here
+function createEmployeeRecord(array) {
+  let employeeObj = {
+    firstName: array[0],
+    familyName: array[1],
+    title: array[2],
+    payPerHour: array[3],
+    timeInEvents: [],
+    timeOutEvents: []
+  };
+  return employeeObj;
+}
+
+function createEmployeeRecords(array) {
+  return array.map(createEmployeeRecord)
+}
+
+function createTimeInEvent(object, datestamp) {
+  let timeInfo = datestamp.split(' ')
+  let date = timeInfo[0]
+  let hour = parseInt(timeInfo[1], 10)
+
+  let timeEvent = {
+    type: 'TimeIn',
+    date: date,
+    hour: hour
+  }
+  
+ object.timeInEvents.push(timeEvent)
+ return object
+}
```

Comment:
Just some minor refactors:
```suggestion
function createTimeInEvent(object, datestamp) {
  let [date, hour] = datestamp.split(' ')
  hour = parseInt(hour, 10)

  let timeEvent = {
    type: 'TimeIn',
    date,
    hour
  }
  
  object.timeInEvents.push(timeEvent)
  return object
}
```